attacks.py

```diff
@@ -1,4 +1,5 @@
 import math
+from arcade import play_sound, load_sound
```

Comment:
This is probably fine to start out, but we may want to create a sound manager class or something. That doesn't have to be in this commit, though.